docs_src/examples/Ch-ExamplesAddingModbusDevice.md

```diff
@@ -1,51 +1,62 @@
 # Modbus
 
-EdgeX - Delhi Release
+EdgeX - Ireland Release
 
-PowerScout 3037 Power Submeter
+This page describes how to connect Modbus devices to EdgeX. In this example, we simulate the [temperature sensor](https://www.audon.co.uk/ethernet_sensors/NANO_TEMP.html) instead of using a real device. This provides a straightforward way to test the device service features.
 
-<https://shop.dentinstruments.com/products/powerscout-3037-ps3037>
-<https://www.dentinstruments.com/hs-fs/hub/472997/file-2378482732-pdf/Pdf_Files/PS3037_Manual.pdf>
-
-![PowerScout 3037 Power Submeter](powerscout.png)
-
-In this example, we simulate the PowerScout meter instead of using a
-real device. This provides a straight-forward way to test the
-device-modbus features.
-
-![Modbus Simulator](simulator.png)
+- Temperature sensor: https://www.audon.co.uk/ethernet_sensors/NANO_TEMP.html
+- User manual: http://download.inveo.com.pl/manual/nano_t/user_manual_en.pdf
 
```

Comment:
Add an **Important Notice** section here.

## Important Notice
To fulfill the [issue #61](https://github.com/edgexfoundry/device-modbus-go/issues/61), there is an important incompatible change after v2 (Ireland release).  In the Device Profile attributes section, the `startingAddress` becomes an integer data type and zero-based value.  In v1, `startingAddress` is a string data type and one-based value.

Reply:
Added the Important Notice.